src/boss/mar/CA94.c

```diff
@@ -0,0 +1,189 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+#include <stage.h>
+
+extern u16 D_80180A60[];
+extern u8 D_us_80180690[];
+extern u8 D_us_801806A4[];
+extern u8 D_us_801806B8[];
+extern u8 D_us_801806CC[];
+extern u8 D_us_801806E4[];
+extern u8 D_us_801806EC[];
+extern u8 D_us_80180700[];
+extern u8 D_us_80180718[];
+extern u8 D_us_80180724[];
+extern u8 D_us_80180738[];
+extern u32 g_CutsceneFlags;
+extern s32 D_us_8019AE70;
+
+void func_us_8018CA94(Entity* self) {
+    Entity* entities;
+    Tilemap* tilemap;
+    s32 pan;
+
+    tilemap = &g_Tilemap;
+    entities = &PLAYER;
```

Comment:
Any reason to call this "entities" rather than "player"?

Also might be worth a comment mentioning that this pointer is unused. Otherwise, looks good.

Reply:
It doesn't seem like this change was made?